docs/INSTALL.K8S.md

```diff
@@ -0,0 +1,41 @@
+Installing Kubernetes
+=====================
```

Comment:
I will reference an actual doc. The last time I read from there, it took me half a week to get a basic k8s installation working as the documentation is ugly with all sort of complications making many assumptions about what user wants. I will keep the quick start for people to get started and hopefully we can get rid of it eventually.